cpp/ql/lib/semmle/code/cpp/controlflow/internal/ConstantExprs.qll

```diff
@@ -110,8 +110,8 @@ private predicate loopConditionAlwaysUponEntry(ControlFlowNode loop, Expr condit
  * should be in this relation.
  */
 pragma[noinline]
-private predicate isFunction(@element el) {
-  el instanceof @function
+private predicate isFunction(Element el) {
+  el instanceof Function
```

Comment:
I think these changes should just be kept.

Reply:
Fixed in fe2c806c2db5f3dceaf2877f34a25fcbce34337b